### PR TITLE
Update default button text color in dark mode

### PR DESCRIPTION
**Pull request by yyorkchan**

### CODE REVIEW

File: authui/src/authflowv2/base.css [hunk 1]
```diff
@@ -101,6 +101,9 @@
 
     /* background */
     --color-bg-base: #1c1c1e;
+
+    /* text */
+    --color-text-base: #ffffff;
```
Comment (louischan-oursky) [inline authui/src/authflowv2/base.css:106]: 1. Remove color-bg-base
2. Add color-surface-primary
3. Add color-white
4. Add color-black
5. Change existing places to use color-surface-primary or color-white, on your own judgement.

Comment (tung2744) [reply authui/src/authflowv2/base.css:106]: I think existing places which uses `--color-bg-base` should point to `color-white` or `color-neutral-700`. `color-surface-primary` and `color-white` is not on the same level, and I prefer not to introduce `color-surface-primary`

Comment (fungc-io) [reply authui/src/authflowv2/base.css:106]: idk if this is relevant, the text button for light and dark mode can be set separately in the future.
image

File: authui/src/authflowv2/base.css [hunk 2]
```diff
@@ -63,7 +63,11 @@
     --color-neutral-700: #131315;
 
     /* background */
-    --color-bg-base: #ffffff;
+    --color-surface-primary: #ffffff;
```
Comment (louischan-oursky) [inline authui/src/authflowv2/base.css:66]: This should be defined as

```
--color-surface-primary: var(--color-white);
```

File: authui/src/authflowv2/base.css [hunk 3]
```diff
@@ -100,10 +104,7 @@
     --color-neutral-700: #1c1c1e;
 
     /* background */
-    --color-bg-base: #1c1c1e;
-
-    /* text */
-    --color-text-base: #ffffff;
+    --color-surface-primary: #1c1c1e;
```
Comment (louischan-oursky) [inline authui/src/authflowv2/base.css:107]: This should be defined as

```
--color-surface-primary: var(--color-neutral-700);
```

File: authui/src/authflowv2/components/tertiary-button.css [hunk 4]
```diff
@@ -1,7 +1,7 @@
 @layer components {
   :root {
     --tertiary-btn__bg-color: var(--color-neutral-700);
-    --tertiary-btn__text-color: var(--color-bg-base);
+    --tertiary-btn__text-color: var(--color-neutral-700);
```
Comment (louischan-oursky) [inline authui/src/authflowv2/components/tertiary-button.css:4]: This should be `--color-white`

File: authui/src/authflowv2/base.css (hunk 2 above)
Comment (louischan-oursky) [reply authui/src/authflowv2/base.css:66]: @yyorkchan Move this below `color-white` and `color-black`, and address my previous comment.

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**